.eslintrc.json

```diff
@@ -46,7 +46,7 @@
           "no-use-before-define": 0,
           "no-tabs": "off",
           "no-console":["error",{"allow":["warn","error"]}],
-          "prettier/prettier": "error",
+          "prettier/prettier": "warn",
```

Comment:
We should discuss this kind of change before, however Son and me are ok with this change